src/tox/config/__init__.py

```diff
@@ -798,6 +798,20 @@ def develop(testenv_config, value):
 
     parser.add_testenv_attribute_obj(DepOption())
 
+    parser.add_testenv_attribute(
+        name="interrupt_timeout",
+        type="float",
+        default=0.3,
```

Comment:
shouldn't the defaults come from the sessions constants?

Reply:
I used the constants here.